### ISSUE TITLE
FieldTextInput editing and zoom in problems

### ISSUE THREAD

**Issue by jlaurens** [google/blockly#1406]
### Problem statement

- [x] Bug report
- [ ] Feature request

When editing a text field, the view port scrolls to the top left eventually hiding the block to be editing.

### Expected Behavior

What happens when no zoom has been applied.

### Actual Behavior

If the zoom factor is big, the block will be out of the viewport.

### Steps to Reproduce

1. open .../demos/fixed/index.html
2. drag a text block to the workspace
3. zoom in (many times depending on your screen size)
4. edit the block

### Stack Traces

NA

### Operating System and Browser

- [ ] Desktop:
  - [X] Chrome
  - [X] Firefox
  - [X] Safari
  - [ ] Opera
  - [ ] IE 10+
  - [ ] IE 11
  - [ ] EDGE

- [ ] Smartphone/Tablet/Chromebook (please complete the following information):

### Additional Information

Most probably a related problem on chrome and safari, but not on firefox:

1. open .../demos/fixed/index.html
2. drag a text block to the workspace
3. zoom in (many times depending on your screen size)
4. move the workspace
5. edit the block

Then the WidgetDiv is not located above the block.

NA

**Comment by rachel-fenichel** [comment]
I'm not seeing this problem on chrome or firefox at https://blockly-demo.appspot.com/static/demos/fixed/index.html

When you said "zoom in", I used browser zoom.  I zoomed to 500%, then clicked on the text to edit it.  The editor widget was in the correct place.

Can you send a screen capture to help me debug this?

**Comment by jlaurens** [comment]
I don't know exactly how I came to that conclusion but now that I have reinstalled many things I cannot reproduce it so I must be wrong. Actually I do have a problem in that respect with Chrome, Safari and FireFox working as expected on one hand and Chromium doing unexpected things on the other. If I am able to narrow the problem, it will merely concern Chromium so let us close this issue and sorry for the disturbance.
